aikau/src/main/resources/alfresco/services/ActionService.js

```diff
@@ -789,14 +771,11 @@ define(["dojo/_base/declare",
        * function.
        *
        * @instance
-       * @param {object} payload The event payload
-       * @param {object} document The document edit offline.
+       * @param {object} payload The payload from the original action request
        */
-      onActionEditOffline: function alfresco_services_ActionService__onActionEditOffline(payload, document) {
-
+      onActionEditOffline: function alfresco_services_ActionService__onActionEditOffline(payload) {
          // Document might be an array.
-         document = (lang.isArray(document))? document[0] : document;
-
+         var document = (lang.isArray(payload.document))? payload.document[0] : payload.document;
```

Comment:
`dojo/_base/lang`'s isXXX methods are all deprecated ... more info on the [dojo/_base/lang page](http://dojotoolkit.org/reference-guide/1.10/dojo/_base/lang.html#deprecated-methods)